zedd-app/src/components/AppGui.tsx

```diff
@@ -99,6 +99,28 @@ export const AppGui = observer(
       return () => window.removeEventListener('keydown', undoAndRedo)
     }, [state])
 
+    useEffect(() => {
+      const clearMarking = (e: KeyboardEvent) => {
+        if (e.key === 'Escape') {
+          state.clearMarking()
+        }
+      }
+      window.addEventListener('keydown', clearMarking)
+      return () => window.removeEventListener('keydown', clearMarking)
+    }, [state])
+
+    useEffect(() => {
+      const removeSlicesOnDelete = (e: KeyboardEvent) => {
+        if (e.key === 'Delete') {
```

Comment:
you can summarize the if conditions

if (e.key === 'Delete' && state.markedSlices.length !== 0)